Utility.cs

```diff
@@ -96,7 +97,7 @@ public static partial class Utility {
 			}
 
 			try {
-				equal = TagIOSave(item1).SequenceEqual(TagIOSave(item2));
+				equal = TagIOSave(item1, savedItemTagIO).Equals(TagIOSave(item2, savedItemTagIO));
```

Comment:
use `string == string` rather than `string.Equals(string)` for better code clarity